dxr/plugins/clang/filters.py

```diff
@@ -100,9 +115,9 @@ class CallerFilter(_CQualifiedNameFilter):
     name = 'callers'
     description = Markup('Functions which call the given function or method: <code>callers:GetStringFromName</code>')
 
-    def __init__(self, term):
+    def __init__(self, term, enabled_plugins):
```

Comment:
I'll go with enabled_plugins, since that's what they are. We can rename it if we later decide to do things differently.